schemas/CreateChargeInput.json

```
@@ -0,0 +1,80 @@
+{
+  "type": "object",
+  "additionalProperties": false,
+  "required": ["original_amount_cents", "merchant_data"],
+  "properties": {
+    "line_item_status": {
+      "type": "string",
+      "description": "the corresponding Status for a line item",
+      "enum": ["VALID", "INVALID", "OFFSET", "PENDING", "AUTHORIZED", "DECLINED", "VOID", "POSTED"],
+      "example": "VALID",
+      "default": "VALID"
+    },
+    "original_amount_cents": {
+      "type": "integer",
+      "description": "The Integer Amount of the Charge in Cents",
+      "example": 200,
+      "default": 0
+    },
+    "effective_at": {
+      "type": "string",
+      "description": "The `Date-Time` that the charge should be recorded to the account. Although the charge is being added to Canopy today, it can actually be effective as of 10 days ago.",
```

```suggestion
      "description": "The `Date-Time` that the charge is made applicable to the account. Although a charge may be made today, it can be set effective 10 days prior.",
```

is this better? especially debatable suggestion